windows/src/test/inffiletests/packageinf.pas

```diff
@@ -0,0 +1,32 @@
+unit packageinf;
+
+interface
+
+procedure run;
+
+implementation
+
+uses
+  System.Win.ComObj,
+  Winapi.ActiveX,
+  PackageInfo,
+  kmpinffile;
+
+procedure run;
+begin
+  CoInitializeEx(nil, COINIT_APARTMENTTHREADED);
+  with TKMPInfFile.Create do
+  try
+    FileName := 'c:\temp\kmp.inf';
+    LoadIni;
+    FileName := 'c:\temp\kmp.xml';
+    SaveXML;
+    FileName := 'c:\temp\kmp.json';
+    SaveJSON;
```

Comment:
Is `c:\temp\` guaranteed to exist? What about using `System.IOUtils.TPath.GetTempPath()`?

Reply:
This is a test app for experimenting with inf files. I saved it just for posterity; we don't really need it (perhaps I could take it out of the repo). In any case, we don't really need to spend time on it.